File: flipt-client-java/src/main/java/io/flipt/client/models/Flag.java
package io.flipt.client.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import lombok.Builder;
import lombok.Value;

@Value
@Builder
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonDeserialize(builder = Flag.FlagBuilder.class)
public class Flag {
  @JsonProperty("key")
  String key;

  @JsonProperty("enabled")
  boolean enabled;

  @JsonProperty("type")
  String type;

  @lombok.experimental.SuperBuilder
  @JsonIgnoreProperties(ignoreUnknown = true)
  public static class FlagBuilder {}
}

File: flipt-client-kotlin-android/src/main/java/io/flipt/client/models/Flag.kt
package io.flipt.client.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Flag(
    @SerialName("key") val key: String,
    @SerialName("enabled") val enabled: Boolean,
    @SerialName("type") val type: String
)